src/components/DetailViewHeader.jsx

```diff
@@ -37,26 +49,38 @@ export class DetailViewHeader extends React.PureComponent {
     addSteward: PropTypes.func,
     canReadPolicies: PropTypes.bool,
     classes: PropTypes.object.isRequired,
+    dispatch: PropTypes.func.isRequired,
+    isProcessing: PropTypes.bool,
```

Comment:
Looks like you need a sort here

Reply:
Do you mean that these need to be sorted alphabetically? (just want to make sure I'm fixing the thing you're referencing)

Reply:
exactly